<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>561</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>770</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>BEFORE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AFTER</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>20</width>
     <height>601</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>561</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>Select_Image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1168</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>1194</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1040</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>1012</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>Slider(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>787</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Select_Image()</slot>
  <slot>Exit()</slot>
  <slot>Save()</slot>
  <slot>Slider(int)</slot>
 </slots>
</ui>
